openwisp_radius/base/models.py

```diff
@@ -1174,8 +1174,10 @@ class AbstractOrganizationRadiusSettings(UUIDModel):
         null=True,
         blank=True,
         help_text=_PASSWORD_RESET_URL_HELP_TEXT,
-        default=PASSWORD_RESET_URLS.get('default'),
-        fallback=PASSWORD_RESET_URLS.get('default'),
+        default=PASSWORD_RESET_URLS.get('__all__')
+        or PASSWORD_RESET_URLS.get('default'),
+        fallback=PASSWORD_RESET_URLS.get('__all__')
+        or PASSWORD_RESET_URLS.get('default'),
```

Comment:
why instead of doing this a zillion times we don't just do it once in the app settings file and call just that?

```python
DEFAULT_PASSWORD_RESET_URL = (
    PASSWORD_RESET_URLS.get('__all__')
    # kept for backward compatibility, will be removed in future versions
    or PASSWORD_RESET_URLS.get('default')
)
```